File: 1588-sum-of-all-odd-length-subarrays/1588-sum-of-all-odd-length-subarrays.c
int sumOddLengthSubarrays(int* arr, int arrSize) {
    int ans=0;
        for(int i=0;i<arrSize;i++)
        {
            int sum=0;
            for(int j=i;j<arrSize;j++)
            {
                sum+=arr[j];
                if((j-i+1)%2!=0)
                {
                    ans+=sum;
                }
            }
        }
        return ans;
}

File: 1588-sum-of-all-odd-length-subarrays/1588-sum-of-all-odd-length-subarrays.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        ans=0
        n=len(arr)
        for i in range(n):
            summ=0
            for j in range(i,n):
                summ+=arr[j]
                if((j-i+1)%2!=0):
                    ans+=summ
        return ans